pyproject.toml

```diff
@@ -27,16 +27,29 @@ readme = "README.md"
 simplesecurity = 'simplesecurity:cli'
 
 [tool.poetry.dependencies]
-python = "^3.7"
-poetry = { version = "<2,>=1.1.10", optional = true }
-bandit = { version = "<2,>=1.7.0", optional = true }
-safety = { version = "<2,>=1.10.3", optional = true }
-dodgy = { version = "<2,>=0.2.1", optional = true }
-dlint = { version = "<2,>=0.11.0", optional = true }
-semgrep = { version = "<2,>=0.67.0", optional = true }
+python = "^3.8"
+poetry = "^1.1.10"
+black = "^22.6.0"
+flake8 = "^4.0.1"
+flake8-bandit = "^3.0.0"
+flake8-json = "^21.7.0"
+bandit = "^1.7.0"
+safety = "^2.2.3"
+dodgy = "<2,>=0.2.1"
+dlint = "<2,>=0.11.0"
+semgrep = "^1.2.0"
+sphinx = "^5.3.0"
+docutils = "0.17"
+sphinx-autodoc-typehints = "^1.19.5"
```

Comment:
Lots of sphinx related deps which is part of my reluctance to include. More versions to manage etc

Reply:
So I did split these dependencies out in the pyproject.toml, so that installation is not always neccesary.